resources/ext/config/conf.d/puppetserver.conf

```diff
@@ -2,13 +2,13 @@
 jruby-puppet: {
     # This setting determines where JRuby will look for gems.  It is also
     # used by the `puppetserver gem` command line tool.
-    gem-home: /var/lib/puppet/jruby-gems
+    gem-home: /opt/puppetlabs/puppet/cache/jruby-gems
 
     # (optional) path to puppet conf dir; if not specified, will use the puppet default
     master-conf-dir: /etc/puppetlabs/agent
```

Comment:
Is this right?  Should this be `/etc/puppetlabs/puppet` like https://github.com/puppetlabs/puppet-specifications/blob/master/file_paths.md shows for puppet-agent confdir?  Not sure if we decided to have a different confdir for the server than what the agent would use?